doc/source/whatsnew/v0.21.0.txt

```diff
@@ -138,6 +138,14 @@ Backwards incompatible API changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 
+.. _whatsnew_0210.api_breaking.Period:
+
+pd.Period objects are now immutable
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:class:`Period` will now raise an ``AttributeError`` when a user tries to assign a new value to the `ordinal` or `freq` attributes (:issue:`17116`).
```

Comment:
Still not fully what he meant :-) 

The "Backwards incompatible API changes" section starts with subsections for things that we think are important enough to explain in more than a single line, and at the end there is section "other API changes" with a bunch of single entries:

https://github.com/jbrockmendel/pandas/blob/19567c8f0de212dec15e17dc918e6eada6a8c354/doc/source/whatsnew/v0.21.0.txt#L280-L295

So you can put it there.